conjure-java-core/src/main/java/com/palantir/conjure/java/util/ErrorGenerationUtils.java

```diff
@@ -63,7 +63,16 @@ public static String errorTypesClassName(ErrorNamespace namespace) {
         return namespace.get() + "Errors";
     }
 
-    public static String errorParametersClassName(String errorName) {
+    public static String serializableErrorClassName(String errorName) {
+        return errorName + "SerializableError";
+    }
+
+    public static String errorExceptionClassName(String errorName) {
+        return errorName + "Exception";
+    }
```

Comment:
nit: for consistency
```suggestion
    public static String serializableErrorClassName(ErrorDefinition errorDefinition) {
        return errorDefinition.getErrorName().getName() + "SerializableError";
    }

    public static String errorExceptionClassName(ErrorDefinition errorDefinition) {
        return errorDefinition.getErrorName().getName() + "Exception";
    }
```